packages/api/src/routes/webhook/tenovi.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import { asyncHandler } from "../util";
+import { tenoviMeasurementSchema } from "../../mappings/tenovi";
+import { processMeasurementData } from "../../command/webhook/tenovi";
+
+const routes = Router();
+
+/** ---------------------------------------------------------------------------
+ * POST /webhook/tenovi
+ *
+ * Receive and process Tenovi Measurements
+ *
+ */
+routes.post(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    processMeasurementData(tenoviMeasurementSchema.parse(req.body));
```

Comment:
How's auth being done? I don't see an auth lambda of sorts, and it seems its possible to have "Header-based authentication":

> Header-based authentication methods (e.g. Basic Auth) can be optionally included in the POST request, if required by the Client.

Reply:
This is something I was going to integrate next in a follow-up update. Could probably let the customer test the integration without auth?

Reply:
Sounds good, let's ticket it so we don't forget it then - or add a checklist on the original ticket w/ one item specifically about this, please.

Reply:
Done!

Reply:
what's the link to the ticket? 😄